src/openthaigpt_pretraining/utils/data_cleaning.py

```diff
@@ -0,0 +1,10 @@
+
+
+def clean_text(line: str) -> str:
```

Comment:
The original code we remove \n and () because we want to train Bert for “discriminative” task. IN GPT case, we snould not remove these all.

Reply:
OK i remove it